src/php/tests/unit_tests/CallTest.php

```diff
@@ -24,12 +24,14 @@ class CallTest extends PHPUnit_Framework_TestCase
     public static function setUpBeforeClass()
     {
         self::$server = new Grpc\Server([]);
-        self::$port = self::$server->addHttp2Port('0.0.0.0:0');
+        self::$port = self::$server->addHttp2Port('0.0.0.0:53000');
```

Comment:
This seems wrong - how do we guarantee there won't be port collisions if you hardcode port 53000?